core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -274,13 +316,17 @@ public JsonObject getAdapterConfiguration(final String type) {
      *                              copied into a new list in order to prevent modification
      *                              of the list after this method has been invoked.
      * @return This tenant for command chaining.
+     * @deprecated Use {@link #setAdapters(List adapters)} instead.
      */
+    @Deprecated
```

Comment:
IMHO the `@JsonProperty` annotation should be moved to either the *setAdapters* method or the *adapters* field ...